src/libraries/Common/src/System/CodeDom/CodeTypeReference.cs

```diff
@@ -327,7 +327,7 @@ public string BaseType
 
                 string returnType = _baseType;
                 return _needsFixup && TypeArguments.Count > 0 ?
-                    returnType + '`' + TypeArguments.Count.ToString(CultureInfo.InvariantCulture) :
+                    $"{returnType}`{(uint)TypeArguments.Count}" :
```

Comment:
@stephentoub, just curious, has the default of string interpolation changed from current culture to InvariantCulture?

Reply:
No.  But this is a non-negative integer.  It's default formatting is unaffected by culture.